detox/test/src/Screens/ExpectationsScreen.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
```

Comment:
So, this is pretty great overall, yet I wonder whether it ought to be better to reuse the 3 input fields we already have in the main actions screens. Wdyt? In essence, this separation is a great initiative, but we should probably either skip it, or do it all the way (i.e. with transferring all other expectations here as well, which isn't necessarily possible)

Reply:
I do think it will be great to split things up.. At least from this point. To put it in the actions screen (which is already loaded with stuff) does not make sense to me.. Naming and functionality

Reply:
Well for now it looks that the new screen only contains 2 input fields, so I'm not sure why the actions screen is insufficient...